index.html

```diff
@@ -168,6 +181,85 @@ <h2>Profesional Art Printing Data</h2>
             </hrx>    
             <button type="button">See Project</button>
           </hrx>
+
+          <hrx class="work-cards-desk img-placeholder2">              
+            <button id = "button-width" type="button">See Project</button>
+          </hrx>
+
+          <hrx class="work-cards-desk img-placeholder4">
+            <h2>Data Dashboard <br> Healthcare</h2>
+            <p>A daily selection of privately personalized reads; 
+              no accounts or sign-ups required. 
+              has been the industry's standard</p>
+            
+            <hrx class="tags">
+              <ul>
+                <li>HTML</li>
+                <li>Bootstrap</li>
+                <li>Ruby</li>
+              </ul>
+            </hrx>    
+          </hrx>
+
+          <hrx class="work-cards-desk img-placeholder3">
+            <h2>Website Protfolio</h2>
+            <p>A daily selection of privately personalized reads; 
+              no accounts or sign-ups required. 
+              has been the industry's standard</p>
+            
+            <hrx class="tags">
+              <ul>
+                <li>HTML</li>
+                <li>Bootstrap</li>
+                <li>Ruby</li>
+              </ul>
+            </hrx>    
+          </hrx>
+
+             <hrx class="work-cards-desk img-placeholder5">
+            <h2>Profesional Art Printing Data More</h2>
+            <p>A daily selection of privately personalized reads; 
+              no accounts or sign-ups required. 
+              has been the industry's standard</p>
+      
+            <hrx class="tags">
+              <ul>
+                <li>HTML</li>
+                <li>Bootstrap</li>
+                <li>Ruby</li>
+              </ul>
+            </hrx>    
+          </hrx>
+
+          <hrx class="work-cards-desk img-placeholder4">
+            <h2>Data Dashboard <br> Healthcare</h2>
+            <p>A daily selection of privately personalized reads; 
+              no accounts or sign-ups required. 
+              has been the industry's standard</p>
+            
+            <hrx class="tags">
+              <ul>
+                <li>HTML</li>
+                <li>Bootstrap</li>
+                <li>Ruby</li>
+              </ul>
+            </hrx>    
+          </hrx>
+
+          <hrx class="work-cards-desk img-placeholder3">
+            <h2>Website Protfolio</h2>
+            <p>A daily selection of privately personalized reads; 
+              no accounts or sign-ups required. 
+              has been the industry's standard</p>
+            
+            <hrx class="tags">
+              <ul>
+                <li>HTML</li>
+                <li>Bootstrap</li>
+                <li>Ruby</li>
+              </ul>
+            </hrx>
+          </hrx>
```

Comment:
- [ ] If you see the design below then you will find that the `About section is covering the whole width and there is a gap before the `About section. Please implement it. See image below for reference
### Your design
![your](https://user-images.githubusercontent.com/81727988/199896879-123e8d9f-1eb4-49bf-a47a-03cc0b14b9ea.png)

### Figma design
![figma](https://user-images.githubusercontent.com/81727988/199896904-520c08f3-1bce-4020-a2cb-f861e98d5006.png)